### ISSUE TITLE
Adapting TAPAS to other languages

### ISSUE THREAD

**Issue by alxblandin** [google-research/tapas#76]
As descirbed here : #75 For research purposes I wonder how your model can be adapted for an another langage. I mean at which step of training (pre-training, intermediate, fine-tuning) the data needs to be in an other natural language than English. Does the vocab file still changes at each of these steps? Or it's only at the first steps of training? If yes it is possible to do few more steps of pre-training after one of your checkpoint ?

**Comment by thomasmueller-google** [comment]
(Mostly repeating the answer from the other issue, feel free to ask more follow up questions.)

We are using the same vocabulary for all training stages. It's the standard BERT uncased vocabulary. It in general is possible to change the vocabulary in between but it would probably require to discard the current word piece embedding table. This might mean a significant loss in what the model has learned.

In general, I would think that some of the things that the model learns during pre-training should translate to table understanding tasks in other languages. Therefore, I would think that the English models can be a decent starting point for fine-tuning tasks in other languages. That said a model pre-trained on the target language would probably work better.

**Comment by alxblandin** [comment]
@thomasmueller-google 
Is it mandatory to use the create-data process? If I put my data exactly in the the form of the train/dev/test.tsv files and the tables corresponding is it enough to launch a training task correctly?

**Comment by thomasmueller-google** [comment]
Yes, that should work. 
You can create interactions and TF examples and put them into the right place and name them in the right way.
The correct file paths depend on the task, though.

**Comment by eisenjulian** [comment]
Closing this for the time being, let us know if there are any further questions. BTW are you thinking of a concrete academic dataset in another language?
